distribution/docs/src/main/resources/content/_installing/installing-intro.adoc

```diff
@@ -62,9 +60,15 @@ These are the system/environment requirements to configure _prior_ to an install
 |
 
 |===
-*These requirements assume the use of a remote catalog. Extra resources will need to be allocated if the system will also be hosting a local catalog.
 
-**The amount of RAM can be increased to support memory-intensive applications. See <<jvm-memory-configuration, Memory Considerations>>
+[NOTE]
+====
+If installing a test, demo, or development installation, see the <<_quick_install_prerequisites,Quick Start Installation Prerequisites>>.
+====
+
+^*^These requirements assume the use of a remote catalog. Extra resources will need to be allocated if the system will also be hosting a local catalog.
```

Comment:
What kind of resources? RAM? CPU? Disk space? More system administrators?

Reply:
All of the above?

Reply:
I do not know. Who would know? Who  works on this part of the system?

Reply:
It would/could be any/all of the above.